Dockerfile

```diff
@@ -3,13 +3,13 @@
 FROM node:14.19.3-slim AS builder
 
 RUN apt-get update \
-  && apt-get -y --no-install-recommends install openssl libc6
+  && apt-get -y --no-install-recommends install openssl libc6 git ca-certificates
```

Comment:
@esacteksab now that we have a released version of react uswds, is it ok to keep the changes you made? or do we need to revert?

Reply:
revert please. I'm happy to do this here or on another branch. Which would you like?

Reply:
here is best since they haven't been merged in yet!

Reply:
I believe it's done.